### DISCUSSION TITLE
Polydispersity

### DISCUSSION THREAD

**Discussion by andreashlarsen** [andreashlarsen/SAStutorials#5] [Q&A]

**Comment by JeppeBreum** [comment]
On the [Polydispersity tutorial](https://sastutorials.org/Polydispersity/Polydispersity.html) I added a question:

> Question: why is the distribution not "perfect" when the data was simulated?

(In reference to Shape2SAS data fitted in McSAS).

As I'm not 100% sure about the answer myself, I thought about adding a hint page.

Here's my (rough) suggestion for the answer:

The data-file contains quite few points for higher q. Higher q-data determines the small-r features... 

Does this sound reasonable?
